core/webhooks/chat.go

```diff
@@ -10,11 +10,11 @@ func SendChatEvent(chatEvent models.ChatEvent) {
 	// TODO: handle errors here instead of returning
 	switch chatEvent.MessageType {
 	case models.MessageSent:
-		webhookEvent.Type = MessageSent
+		webhookEvent.Type = models.MessageSent
 	case models.UserNameChanged:
-		webhookEvent.Type = UserNameChanged
+		webhookEvent.Type = models.UserNameChanged
 	case models.VisibiltyToggled:
-		webhookEvent.Type = VisibiltyToggled
+		webhookEvent.Type = models.VisibiltyToggled
 	}
 
 	webhookEvent.EventData = &WebhookChatMessage{
```

Comment:
I guess now with the types being the same we could just create webhookEvent directly and just pass MessageType directly over?

Reply:
Yeah I left it just to keep the structure of what you already did, but I'm just going to assign it directly.